src/components/shared/Repository/TagsList.tsx

```diff
@@ -39,6 +37,24 @@ export default function TagsList(props) {
   // TODO: refactor
   const [organization, ...repo] = location.pathname.split('/').slice(2);
   const repository = repo.join('/');
+  const PageBreadcrumbs = [
```

Comment:
can you introduce an interface which defines the breadcrum entry?

Reply:
added in new file